apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php

```diff
@@ -406,6 +414,10 @@ function load_block_description_links() {
  * Load WP.com Global Styles.
  */
 function load_wpcom_global_styles() {
+	// Doesn't work without this function defined, and it's only defined on wpcom.
+	if ( ! function_exists( 'wpcom_get_site_purchases' ) ) {
```

Comment:
These features simply don't work locally with wp-env, since they rely on wordpress.com code. As a result, we disable them so the rest of the plugin can still load locally.

Reply:
Edit: I'm not adding this in this PR; maybe later.